vendor/bat-native-ledger/src/bat/ledger/ledger.cc

```diff
@@ -677,6 +677,96 @@ bool AutoContributeProps::loadFromJson(const std::string& json) {
   return !error;
 }
 
+CurrentReconcileInfo::CurrentReconcileInfo()
```

Comment:
nit: why is this `CurrentReconcileInfo` and not just `ReconcileInfo`?

Reply:
Yeah, that's wrong. I'll fix.